contrib/debian/copyright

```diff
@@ -1,161 +0,0 @@
-Format: http://svn.debian.org/wsvn/dep/web/deps/dep5.mdwn?rev=174
```

Comment:
This is a [required file](https://www.debian.org/doc/manuals/maint-guide/dreq.en.html#copyright).